application/src/main/java/org/thingsboard/server/ThingsboardInstallApplication.java

```diff
@@ -29,6 +29,7 @@
 @ComponentScan({"org.thingsboard.server.install",
         "org.thingsboard.server.service.component",
         "org.thingsboard.server.service.install",
+        "org.thingsboard.server.service.entityprofile",
```

Comment:
Let's rename package to "profile" (for simplicity)

Reply:
👌

Reply:
done